ci/README.md

```diff
@@ -107,3 +108,18 @@ Dependencies are installed by the `ci/mac_ci_setup.sh` script, via [Homebrew](ht
 which is pre-installed on the CircleCI MacOS image. The dependencies are cached are re-installed
 on every build. The `ci/mac_ci_steps.sh` script executes the specific commands that
 build and test Envoy.
+
+# Coverity Scan Build Flow
+
+[Coverity Scan Envoy Project](https://scan.coverity.com/projects/envoy-proxy)
+
+
+Coverity Scan static analysis is not run within Envoy CI. However, Envoy can be locally built and submitted for analysis.
+A Coverity Scan Envoy project token must be generated from the Coverity project settings, [Coverity Project Settings](https://scan.coverity.com/projects/envoy-proxy?tab=pro).
```

Comment:
nit: You can just link "Coverity project settings" directly, vs repeating the text. Also, if you don't mind, can you try to flow the text to about 100 col.